internal/windowsansi/winansi.go

```diff
@@ -0,0 +1,41 @@
+//go:build windows
+
+package windowsansi
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+// Enable activates ANSI support on Windows terminals by setting the
+// ENABLE_VIRTUAL_TERMINAL_PROCESSING flag.
+// It fails silently if the output is not a console.
+func Enable() {
+	const enableVirtualTerminalProcessing = 0x0004
+
+	// Load kernel32.dll and the necessary procedures dynamically.
+	// This avoids a hard dependency and allows the program to run on non-Windows
+	// systems, although this file is guarded by a build tag.
+	kernel32 := syscall.NewLazyDLL("kernel32.dll")
+	procGetConsoleMode := kernel32.NewProc("GetConsoleMode")
+	procSetConsoleMode := kernel32.NewProc("SetConsoleMode")
+
+	// Get the handle for standard output.
+	handle := syscall.Handle(os.Stdout.Fd())
+	var mode uint32
+
+	// GetConsoleMode fails if not in a real console.
+	ret, _, _ := procGetConsoleMode.Call(uintptr(handle), uintptr(unsafe.Pointer(&mode)))
+	if ret == 0 {
+		return
+	}
```

Comment:
The error is ignored, it could be returned via `Enable() error`

It would be cleaner. Even if for now, Enable is called via init that would ignore it.